packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1299,6 +1299,67 @@ test('adjust balances', async t => {
   });
 });
 
+test.only('adjust balances after interest charges', async t => {
```

Comment:
```suggestion
test('adjust balances after interest charges', async t => {
```

Reply:
ty. https://github.com/Agoric/agoric-sdk/issues/4941 is way back burner